--- origsrc/camelia/src/camelia.pro	2005-09-21 10:18:12.000000000 -0500
+++ src/camelia/src/camelia.pro	2007-04-26 03:17:45.390625000 -0500
@@ -10,10 +10,10 @@
 target.path = $$PREFIX/bin
 
 data.path = $$PREFIX/lib/camelia
-data.extra = cp -R ../data/* $${data.path}
+data.extra = cp -Lr ../data/* $${data.path}
 
 icon.path = $$PREFIX/share/pixmaps
-icon.extra = cp ../data/icons/camelia.png $${icon.path}
+icon.extra = cp -L ../data/icons/camelia.png $${icon.path}
 
 
 
